# Goal
float64 distance

---
# Result
bool success

---
# Feedback
float64 remaining_distance
